pkg/plugins/runtime/gateway/route/configurers.go

```diff
@@ -183,7 +180,6 @@ func RouteMatchRegexQuery(name string, regex string) RouteConfigurer {
 
 func RouteAppendHeader(name string, value string) *envoy_config_core.HeaderValueOption {
 	return &envoy_config_core.HeaderValueOption{
-		Append: util_proto.Bool(true),
```

Comment:
maybe we should always include `AppendAction`? It's not clear to me what the logic is when I see this..

Reply:
done